java-checks/src/test/files/checks/SynchronizedOverrideCheck.java

```diff
@@ -0,0 +1,21 @@
+class A {
+  public synchronized void f1(){}
+  public void f2(){}
+  public synchronized void f3(){}
+}
+
+class B extends A{
+  @java.lang.Override
+  public void f1(){} // Noncompliant [[sc=15;ec=17]] {{Make this method "synchronized" to match the parent class implementation.}}
+
+  @java.lang.Override
+  public void f2(){} // Compliant
+
+  public void f3(){} // Noncompliant {{Make this method "synchronized" to match the parent class implementation.}}
+}
+
+class C extends B{
+  @java.lang.Override
+  public void f1(){} // Compliant
```

Comment:
I don't know about this one. I think it deserve an issue as well... because if we don't then the issue will appear only after fixing the first one. Which is OK, but could be painful sometime. Up to you to decide, I let you judge.

Reply:
I agree with you but on another side you may not have access or you may do not see if `synchronized` is written far away in the hierarchy...
Both choices have drawbacks, I think I will keep this one for now.